Makefile

```diff
@@ -0,0 +1,19 @@
+check: check-line-length check-syntax
+
+check-line-length:
+	flake8 ./pybsn/ ./bin/* --count --max-complexity=20 --max-line-length=127 --statistics
+
+check-syntax:
+	flake8 ./pybsn/ ./bin/* --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
@andre-bigswitch, one q: actually, where do those settings come from?

Reply:
Check out this comment: https://github.com/bigswitch/infrastructure/pull/239/files#r577083871